### DISCUSSION TITLE
Auto reboot device

### DISCUSSION THREAD

**Discussion by SPancerfaustyna** [AlexProgrammerDE/balena-minecraft-server#168] [Ideas]
Hey, I would like to have possibility to reboot automatically my device, for example once a day at 2:00am to clear RAM and to make the server work better. Is there possibility right now or can you implement that?

**Comment by viedok** [comment]
There are plugins for that, for example [UltimateAutoRestart](https://www.spigotmc.org/resources/1-8-1-18-3-5k-users-ultimateautorestart-need-an-autorestart-plugin-grab-the-best-one-today.64414/) works great.
